packages/http-server/test/integration/http-server.integration.ts

```diff
@@ -30,7 +30,8 @@ describe('HttpServer (integration)', () => {
       });
 
   it('starts server', async () => {
-    server = new HttpServer(dummyRequestHandler);
+    const serverOptions = process.env.TRAVIS ? {host: '127.0.0.1'} : {};
```

Comment:
Please don't repeat this TRAVIS check in all pull request. Create a helper function for creating server options and move the check there. For example:

```
// usage here
const serverOptions = givenServerOptions();

// example - use non-default settings
const serverOptions = givenServerOptions({protocol: 'https'});

// implementation
function givenServerOptions(options: Partial<HttpServerOptions>): HttpServerOptions {
  const defaults = process.env.TRAVIS ? {host: '127.0.0.1'} : {};
  return Object.assign(defaults, options);
}
```

Reply:
> Please don't repeat this TRAVIS check in all __pull request__.

You mean __test case__?

Reply:
Ah, sorry for the confusion - yes, I meant don't repeat the check in multiple test cases.